packages/fluentui/docs/src/examples/components/Tree/Performance/TreeWith60ListItems.perf.tsx

```diff
@@ -55,6 +55,7 @@ const items = [
 // TreeTitle with just text, while the level 2 titles are rendered as ListItems.
 const titleRenderer = (Component, { content, header, headerMedia, media, ...restProps }) => {
   // as providing all props to List.Item was showing console errors, therefore reducing props
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
pls fix the rule instead: https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings